extensions.md

```diff
@@ -59,6 +59,7 @@ Issues with customtype include:
   * <a href="https://github.com/gogo/protobuf/issues/125">Using a proto message as a customtype is not allowed.</a>
   * <a href="https://github.com/gogo/protobuf/issues/200">cusomtype of type map can not UnmarshalText</a>
   * <a href="https://github.com/gogo/protobuf/issues/201">customtype of type struct cannot jsonpb unmarshal</a>
+  * Custom types are required to implement a handful of methods. See [CustomTypes](custom_types.md) for more information.
```

Comment:
I think we can move all these issues to the customtypes.md file and then simply reference the customtypes.md file in the table above.